src/components/blocks/infoboks/InfoBoksWithImageAndButton.tsx

```diff
@@ -0,0 +1,55 @@
+import Image from "next/image";
+import {BodyLong, Button, Heading, Panel} from "@navikt/ds-react";
+import React from "react";
+import styled from "styled-components";
+
+const StyledPanel = styled(Panel)`
+  margin-bottom: 2rem;
+`;
+
+const PanelContent = styled.div`
+  display: flex;
+  padding: 1rem;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
+  text-align: center;
+`;
+
+interface Props {
+  imageSrc: string;
+  heading: string;
+  description?: string;
+  buttonText: string;
+  show: boolean;
+  onClose(show: boolean): void;
+}
+
+export const InfoBoksWithImageAndButton = ({
+  imageSrc,
+  heading,
+  description,
+  buttonText,
+  show,
+  onClose,
+}: Props) => {
+
+  if (!show){
+    return null
+  }
+  return (
+        <StyledPanel border={true}>
+        <PanelContent>
```

Comment:
Hm, ser kanskje ut som prettier ikke er kjørt her? Last gjerne ned "prettier" plugin til intellij for å formattere "live", eller så kan du kjøre npm run prettier for å formattere alle filer som gjenstår i prosjektet.

Luan har også en PR som auto-kjører prettier på alle endringer, men denne er ikke i prod enda :)